plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -0,0 +1,136 @@
+class BinaryBuilder
+  DOCKER_BUILD_FILE = 'Dockerfile.build'.freeze
+  BUILD_SCRIPT = '/app/build.sh'.freeze
+  ARTIFACTS_FILE = 'artifacts.tar'.freeze
+  ARTIFACTS_FILE_PATH = "/app/#{ARTIFACTS_FILE}".freeze
+  DOCKER_HOST_CACHE_DIR = '/opt/samson_build_cache'.freeze
```

Comment:
I'm with @steved on this one.  Having a single cache directory that all projects share sounds like it could cause cross-project contamination. Like if you ran build scripts that compiled a bunch of .java files into .class files, and then added `*.class` into your archive, but whoops, that just included a bunch of object files from some completely different project.

+1 on having a cache directory per project.  (Ideally, there would be some mechanism to clean up that directory, but we can make that an enhancement later.)